app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -57,25 +56,31 @@ class PocketVideoFragment : Fragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        fun displayFeed(layout: View, videos: List<PocketVideo>) {
-            layout.videoFeed.gridView.adapter = PocketVideoAdapter(context!!, videos, fragmentManager!!)
+        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+
+        fun initAdapter(): PocketVideoAdapter {
```

Comment:
nit: I don't feel this improves readability much over doing it in-line like:
```kotlin
val adapter = PocketVideoAdapter(context!!, fragmentManager!!)
layout.videoFeed.gridView.adapter = adapter
```

Or maybe (is this how `also` is supposed to be used?)
```kotlin
val adapter = PocketVideoAdapter(context!!, fragmentManager!!).also {
    layout.videoFeed.gridView.adapter = it
}